airbyte-integrations/connectors/source-nytimes/source_nytimes/spec.yaml

```diff
@@ -0,0 +1,44 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/nytimes
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Nytimes Spec
+  type: object
+  required:
+    - api_key
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      title: API Key
+      description: API Key
+      airbyte_secret: true
+      order: 0
+    year:
+      type: integer
+      title: Year
+      description: Year
+      minimum: 1851
+      order: 1
+    month:
+      type: integer
+      title: Month
+      description: Month
+      minimum: 1
+      maximum: 12
+      order: 2
```

Comment:
I think this must be changed to `start_date` and implement incremental syncs. Let me know if you need help here.

Reply:
I agree that this would be a good idea. I have been looking at `datetime_stream_slicer.py` and it seems that it does not support monthly increments. I think it would be good for this use case. I think that I'm able to contribute that change as well.